jaxrs-clients/src/main/java/com/palantir/remoting/jaxrs/Client.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.remoting.jaxrs;
+
+import com.palantir.config.service.ServiceConfiguration;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Static factory methods for producing creating HTTP proxies.
+ */
+public final class Client {
+
+    private Client() {}
+
+    /** Creates a client for the given service configuration. The user agent */
+    public static <T> T create(Class<T> serviceClass, String userAgent, ServiceConfiguration config) {
+        ClientBuilder client = jaxrs().ssl(config.security());
+        if (config.connectTimeout().isPresent()) {
```

Comment:
Since `ClientBuilder` converts back to a `Duration` maybe the API should just be a dropwizard `Duration`? Alternatively, we can shade that class (probably should've vs. taking an explicit DW dependency).